src-ts/tools/learn/learn-lib/course-outline/collapsible-item/CollapsibleItem.tsx

```diff
@@ -30,6 +30,8 @@ interface CollapsibleItemProps {
 const CollapsibleItem: FC<CollapsibleItemProps> = (props: CollapsibleItemProps) => {
     const [isOpen, setIsOpen]: [boolean, Dispatch<SetStateAction<boolean>>] = useState<boolean>(false)
 
+    const isAssessment = props.lessonsCount === 1;
```

Comment:
@vas3a I'm thinking we might want to modify the course metadata when we import it to explicitly call out assessments rather than rely on just a single lesson existing in a module. Thoughts?

Reply:
@testflyjets that would be great! This can be easily updated once we have the proper metadata, sure!
For now I think it's ok like this?